src/doc/dev_manual/PrepareRelease.rst

```diff
@@ -153,6 +153,22 @@ is only available on the Mac. ::
     cd ..
     iconutil --convert icns VisItIcon.iconset
 
+Creating a new release notes file
+"""""""""""""""""""""""""""""""""
```

Comment:
Should there be a blank line here? I'm not sure it's necessary, but I always add one.

Reply:
Patch release notes should go on the RC and minor release notes should go on develop. I think they should always create the next patch release file assuming there will be another patch. It there isn't one, the release notes from it can be incorporated into the minor release notes file. If they are finishing a minor release, they should create the files for the new minor release and the new patch release.